File: go/wam/7166.go
const WamDyiReportRequest = 7166 // channel: regular

type DyiReportRequestEvent struct {
	DyiReportType DYIREPORTTYPECODE `wam:"dyi_report_type"` // field 1
	DyiTriggerType DYITRIGGERTYPECODE `wam:"dyi_trigger_type"` // field 2
	DyiRequestErrorMessage string `wam:"dyi_request_error_message"` // field 3
	DyiRequestSuccess bool `wam:"dyi_request_success"` // field 4
}

File: ts/wam/7166.ts
export const WamDyiReportRequest = 7166 as const

export interface DyiReportRequestEvent {
  /** field 1, wire `dyi_report_type` */
  dyiReportType?: DYIREPORTTYPECODE
  /** field 2, wire `dyi_trigger_type` */
  dyiTriggerType?: DYITRIGGERTYPECODE
  /** field 3, wire `dyi_request_error_message` */
  dyiRequestErrorMessage?: string
  /** field 4, wire `dyi_request_success` */
  dyiRequestSuccess?: boolean
}